Assets/MixedRealityToolkit.Extensions/SpectatorView/Scripts/Calibration/CalibrationAPI.cs

```diff
@@ -0,0 +1,389 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Extensions.PhotoCapture;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Extensions.Experimental.SpectatorView
+{
+    internal class CalibrationAPI
+    {
+#if UNITY_WSA
+        [DllImport("SpectatorViewPlugin", EntryPoint = "InitializeCalibration")]
+#else
+        [DllImport("SpectatorViewPlugin.Editor", EntryPoint = "InitializeCalibration")]
+#endif
+        internal static extern bool InitializeCalibrationNative();
```

Comment:
>InitializeCalibrationNative [](start = 36, length = 27)

Question about the mechanics of this (and I guess, maybe the answer is I just get pointed to docs on how this works). Is this a "one global per process initialization step" or is it possibly thread dependent as well? Since we do it in response to newing up the CalibrationAPI I asssume that the underlying APIs as well must be thread-safe (either that or whatever it's doing must set things up for the entire process)

Reply:
Hm, subsequent initializations clear the data. We should probably change this to be a static access that has a clear data plugin function compared to muddling the two functionalities.